qemu/tests/x86_cpu_flags.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from avocado.utils import cpu
+from virttest import error_context, env_process
+from qemu.tests.x86_cpu_model import check_flags
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu reboot test:
+    1) Get cpu flags lists supported by host
```

Comment:
Where is the code for this step?

Reply:
it is in the function `check_flags`, update steps.

Reply:
I see, normally this step is not necessary for this case.  It's just a way to check flags in current implementation. The updated step is much better.